### ISSUE TITLE
failed to find module

### ISSUE THREAD

**Issue by k7732** [catgaming/ldb_bypass#1]
←[38;2;64;157;255m[2024-01-31 11-29-58] ←[38;2;255;16;32m[ERR ]←[0m failed to find process
←[38;2;70;155;255m[2024-01-31 11-29-58] ←[38;2;64;64;64m[DBG ]←[0m LockDownBrowser.exe not running. loading...
←[38;2;76;153;255m[2024-01-31 11-29-58] ←[38;2;64;64;64m[DBG ]←[0m successfully created process
←[38;2;82;151;255m[2024-01-31 11-29-58] ←[38;2;64;64;64m[DBG ]←[0m successfully opened process handle
←[38;2;89;96;255m[2024-01-31 11-30-08] ←[38;2;255;16;32m[ERR ]←[0m failed to find module: LockDownBrowser.dll
←[38;2;14;120;255m[2024-01-31 11-30-11] ←[38;2;255;16;32m[ERR ]←[0m failed to create snapshot

i get these any idea how to fix it? lockdown runs but i cant alt + tab. im using Version: 2.1.2.01

also i highly recommend a vmware/sandbox detection bypass instead because youll just need to RET C3 the antivm function

**Comment by catgaming** [comment]
Sorry I didn't get to this issue sooner, I haven't touched this project in a couple days.

I'm not sure why sometimes the tl32snapshot fails... I think I had it happen to me a couple times, but I couldn't reproduce it (and therefore couldn't debug it). I'll leave the issue open for now in case I or you come up with an answer to that. 

But yes, if the module isn't found, no patches are going to be made.. and therefore alt tab won't work.

And btw, sorry about the ansi escape codes not working on whatever console you're using (probably native windows?). I'll push an update to the logger (hopefully) with a check for the console mode. I use Windows Terminal so I don't have any issues with that, and since I made the logger for my projects, it didn't cross my mind to check the capabilities of the terminal.

**Comment by catgaming** [comment]
If this happens every single time you run the program and the console just gets spammed with errors, then you're building on the wrong architecture. I feel like I have to mention it just in case that's your issue lol..

**Comment by DeadFrostt** [comment]
@k7732 did you get it running? I am not able to comple it with either gcc or CLANG.

**Comment by k7732** [comment]
Hello, thank you for the reply. I realized my mistake was in Visual Studio settings. After switching the configuration to 'Release' and platform to 'x86', it worked perfectly. My apologies for initially selecting 'x64'

alt + tab works fine. thank you so much!

**Comment by k7732** [comment]
> @k7732 did you get it running? I am not able to comple it with either gcc or CLANG.

Use visual studio

**Comment by DeadFrostt** [comment]
@k7732 yeah visual studio seems to work fine, did you try this with any quizzes tho? or any exams does it work with that? lmk if you have, thanks.

**Comment by catgaming** [comment]
It most likely will NOT work with any quizzes. I believe they have a check (WinVerifyTrust I think since the main module imports it) for dll modifications. And since I'm just doing byte patches, it'll most likely detect it. I could probably bypass it, but I've been working on another project of mine. I've been kinda ignoring this project since my quizzes aren't.. difficult

**Comment by DeadFrostt** [comment]
> It most likely will NOT work with any quizzes. I believe they have a check (WinVerifyTrust I think since the main module imports it) for dll modifications. And since I'm just doing byte patches, it'll most likely detect it. I could probably bypass it, but I've been working on another project of mine. I've been kinda ignoring this project since my quizzes aren't.. difficult

Ahh thats fair. Thank you for the response tho, I have one class which is absolutely destroying me with the quizzes and some of them are even in class so I cant use another device LOL, no worries tho but if you think about getting back to this project do update it here if you can would help me alot lmao.

**Comment by catgaming** [comment]
I'll take a look at it ig. No promises that I'll find anything (I'm not that good at reversing. I mainly started this project to try and get better). The main issue I'm having is that I can't really do any dynamic analysis (with a debugger). Mainly cuz the ldb window jumps to foreground if it opens a dialog window. I've just had more motivation for my other projects atm..
